BackEnd/profiles/templates/profiles/email_template.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="uk">
+<head>
+    <style>
+        body {
+            color: black;
+            font-family: Arial, sans-serif;
+        }
+        p, b {
+            color: black;
+        }
+        .button {
+            padding: 10px 20px;
+            color: black;
+            text-decoration: none;
+            border: 1px solid black;
+            border-radius: 5px;
+            display: inline-block;
+            font-family: Arial, sans-serif;
+            margin: 5px;
+        }
+        .button-approve {
+            background-color: white;
+        }
+        .button-reject {
+            background-color: white;
+        }
+        .button:hover {
+            background-color: lightgray;
+        }
+        .image-container {
+            margin-top: 10px;
+        }
+        .image-container p {
+            margin: 0;
+            font-family: Arial, sans-serif;
+            color: black;
+        }
+        .image-container img {
+            width: 100px;
+            height: 100px;
+            border-radius: 50%;
+            object-fit: cover;
+            display: block;
+            margin-top: 5px;
+        }
+        .banner {
+            width: 100%;
+            max-width: 1500px;
+            height: 200px;
+            background: lightgray;
+            display: flex;
+            justify-content: center;
+            align-items: center;
```

Comment:
Many email apps do not support flexboxes. For the best compatibility for email template development, tables are recommended to align items. Check it https://www.campaignmonitor.com/css/flexbox/align-content/